### PR TITLE
Story/UI color scheme

### PR DESCRIPTION
**Pull request by NicoleSelig**
Changes:

- Dasboard header as listbox
- new color scheme
- fixed the assets
image

### CODE REVIEW

File: web/components/BarChart.tsx [hunk 1]
```diff
@@ -1,102 +1,68 @@
-import React, { useEffect, useState } from 'react'
-import PropTypes, {InferProps} from 'prop-types'
+import React, { useEffect, useState } from "react";
+import PropTypes, { InferProps } from "prop-types";
 import { Bar } from "react-chartjs-2";
-import { Menu } from '@headlessui/react'
-import type { ChartData, ChartOptions } from 'chart.js';
 // import teamData from "../data/teams"
-import{
```
Comment (NicoleSelig) [inline web/components/BarChart.tsx:7]: remove the commented code

File: web/pages/teams.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import {data} from '../data/teams'
+
+function teams() {
+    // let winrate: number[] = []
+    // let rankings: number[] = []
+    // // let winrate: number[] = []
+    // data.map((teamData) => (
+    //     winrate.push(teamData.wins/(teamData.wins + teamData.loses))
+    // ))
+    // winrate.sort(function(a, b){return b - a})\let rankings: number[] = []
```
Comment (NicoleSelig) [inline web/pages/teams.tsx:13]: remove the commented code here

File: web/pages/teams.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    data.sort(function (a, b) {
+        if(a.wins!=b.wins){
```
Comment (NicoleSelig) [inline web/pages/teams.tsx:15]: Did you mean to use a strict equality operator here? https://www.geeksforgeeks.org/difference-between-and-operator-in-javascript-2/

File: web/pages/teams.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            if(a.wins > b.wins)
+            return -1
+        }
+        return 0
+    });
+    data.sort(function (a, b) {
+        if(a.wins==b.wins){
```
Comment (NicoleSelig) [inline web/pages/teams.tsx:22]: Did you mean to use a strict equality operator here?

File: web/pages/teams.tsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+function teams() {
+    // let winrate: number[] = []
+    // let rankings: number[] = []
+    // // let winrate: number[] = []
+    // data.map((teamData) => (
+    //     winrate.push(teamData.wins/(teamData.wins + teamData.loses))
+    // ))
+    // winrate.sort(function(a, b){return b - a})\let rankings: number[] = []
+    data.sort(function (a, b) {
```
Comment (NicoleSelig) [inline web/pages/teams.tsx:14]: Stylistically, helper functions should be arrow styled
```suggestion
    data.sort((a, b) => {
```

File: web/pages/teams.tsx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        if(a.wins!=b.wins){
+            if(a.wins > b.wins)
+            return -1
+        }
+        return 0
+    });
+    data.sort(function (a, b) {
```
Comment (NicoleSelig) [inline web/pages/teams.tsx:21]:
```suggestion
    data.sort((a, b) => {
```

**Review by Nathanlee29 — APPROVED** [review]

**Pull request merged**